catalogue_pipeline/ingestor/src/test/scala/uk/ac/wellcome/platform/ingestor/services/IngestorWorkerServiceTest.scala

```diff
@@ -184,30 +219,31 @@ class IngestorWorkerServiceTest
                                            actorSystem: ActorSystem,
                                            metricsSender: MetricsSender,
                                            workIndexer: WorkIndexer)(
-    testWith: TestWith[IngestorWorkerService, R]): R = {
-    withLocalSqsQueue { queue =>
-      withLocalS3Bucket { bucket =>
-        withMessageStream[IdentifiedWork, R](
-          actorSystem,
-          bucket,
-          queue,
-          metricsSender) { messageStream =>
-          val elasticConfig = ElasticConfig(
-            documentType = itemType,
-            indexV1name = esIndexV1,
-            indexV2name = esIndexV2
-          )
-
-          val service = new IngestorWorkerService(
-            elasticConfig = elasticConfig,
-            identifiedWorkIndexer = workIndexer,
-            messageStream = messageStream,
-            system = actorSystem
-          )
-
-          testWith(service)
+    testWith: TestWith[(QueuePair, Bucket), R]): R = {
+    withLocalSqsQueueAndDlq {
+      case queuePair @ QueuePair(queue, dlq) =>
+        withLocalS3Bucket { bucket =>
+          withMessageStream[IdentifiedWork, R](
```

Comment:
This fixture feels like a weird hybrid – some things get passed in that are constructor by external fixtures (ActorSystem, MetricsSender), others are defined internally (queuePair, S3 bucket).

We should be consistent – does this fixture do everything, or nothing?